frontend/src/pages/DataCatalog/MethodologyTab.tsx

```diff
@@ -716,16 +716,44 @@ function MethodologyTab() {
                     race/ethnicity.
                   </li>
                   <li>
-                    <b>Asian (Non-Hispanic)</b>: A single race (Asian), not
-                    Hispanic/Latino.
+                    <b>American Indian and Alaska Native (Non-Hispanic)</b>: A
+                    person having origins in any of the original peoples of
+                    North and South America (including Central America) and who
+                    maintains tribal affiliation or community attachment.
+                  </li>
+                  <li>
+                    <b>Asian (Non-Hispanic)</b>: A person having origins in any
+                    of the original peoples of the Far East, Southeast Asia, or
+                    the Indian subcontinent including, for example, Cambodia,
+                    China, India, Japan, Korea, Malaysia, Pakistan, the
+                    Philippine Islands, Thailand, and Vietnam.
+                  </li>
+                  <li>
+                    <b>Black or African American (Non-Hispanic)</b>: A person
+                    having origins in any of the Black racial groups of Africa.
                   </li>
                   <li>
                     <b>Hispanic/Latino</b>: Any race(s), Hispanic/Latino.
                   </li>
                   <li>
-                    <b>Black or African American</b>: A single race (African
-                    American), including those who identify as African American
-                    and Hispanic/Latino.
+                    <b>
+                      Native Hawaiian or Other Pacific Islander (Non-Hispanic)
+                    </b>
+                    : A person having origins in any of the original peoples of
+                    Hawaii, Guam, Samoa, or other Pacific Islands.
+                  </li>
+                  <li>
+                    <b>Two or more races (Non-Hispanic)</b>: Combinations of two
+                    or more of the following race categories: "White," "Black or
+                    African American," American Indian or Alaska Native,"
+                    "Asian," Native Hawaiian or Other Pacific Islander," or
+                    "Some Other Race"
+                  </li>
+                  <li>
+                    <b>Two or more races & Unrepresented race (Non-Hispanic)</b>
```

Comment:
Let's define "unrepresented race" first, using the census definition for "some other race", and then have this combo race that explains it's a combo of these two previously defined races